deploy/kustomize/olm-catalog/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+bases:
+- ./crd # this would be generated using: make operator-generate-crd
+- ../operator
+
+resources:
+- ../../common/pmem-csi.intel.com_v1alpha1_deployment_cr.yaml
+
+images:
+- name: intel/pmem-csi-driver
+  newTag: vX.Y.Z # this should be replaced with real version number
```

Comment:
> I think you mean "this version will be replaced during make operator-generate-crd with the actual version number".

Yes, it will be replaced by `operator-generate-catalog` make target. Corrected the comment.